src/cargo/core/compiler/job_queue.rs

```diff
@@ -264,7 +264,7 @@ impl<'a> JobQueue<'a> {
         }
         let duration = cx.config.creation_time().elapsed();
         let time_elapsed = format!(
-            "{}.{1:.2} secs",
```

Comment:
This is correct, because we divide nanoseconds (10^9) by 10^7, so we always have only one or two digits here